Package@swift-5.3.swift

```diff
@@ -0,0 +1,34 @@
+// swift-tools-version:5.3
+
+import PackageDescription
+
+let pkg = Package(name: "PromiseKit")
+pkg.platforms = [
+   .macOS(.v10_10), .iOS(.v9), .tvOS(.v9), .watchOS(.v2)
+]
+pkg.products = [
+    .library(name: "PromiseKit", targets: ["PromiseKit"]),
+]
+
+let pmk: Target = .target(name: "PromiseKit")
+pmk.path = "Sources"
+pmk.exclude = [
+    "AnyPromise.swift",
+    "AnyPromise.m",
+    "PMKCallVariadicBlock.m",
+    "dispatch_promise.m",
+    "join.m",
+    "when.m",
+    "NSMethodSignatureForBlock.m",
+    "after.m",
+    "hang.m",
+    "race.m",
+    "Deprecations.swift",
+    "Info.plist"
```

Comment:
`Info.plist` has been added to fix the warning:
![image](https://user-images.githubusercontent.com/2575555/99157242-d515e600-26cf-11eb-92de-1dd2ec27db90.png)

Reply:
I guess this warning was due to the swift-tools-version bump from 5.0 to 5.3.